js.go

```diff
@@ -989,41 +1036,41 @@ var (
 type DeliverPolicy int
 
 const (
-	DeliverAll DeliverPolicy = iota
-	DeliverLast
-	DeliverNew
-	DeliverByStartSequence
-	DeliverByStartTime
+	DeliverAllPolicy DeliverPolicy = iota
```

Comment:
Renamed a few of these to be able to use `nats.DeliverAll()` as an option for Subscribe instead.

Reply:
I prefer the shorter versions TBH.

```go
nats.DeliverAll
nats.DeliverNew
nats.DeliverLast
nats.StartTime
nats.StartSequence
```

Reply:
Could also change server to match..

Reply:
Changed to use the order names, this is the current mapping of functional option to delivery policy:

- nats.DeliverAllAvailable(): nats.DeliverAll
- nats.DeliverLastReceived(): nats.DeliverLast
- nats.DeliverNextReceived(): nats.DeliverNew
- nats.DeliverFromTime(publishTime): nats.StartTime
- nats.DeliverFromSequence(6): nats.StartSequence

Reply:
I meant the function names such that things like `js.Subscribe("foo", nats.StartSequence(100))`

That is what I was thinking we should make shorter.. WDYT?

Reply:
Sounds good to me! Just pushed an update using the shorter names for the options.